Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -3,69 +3,186 @@
 
 using UnityEngine;
 using UnityEngine.Events;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
 
-namespace Microsoft.MixedReality.Toolkit.Input
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
```

Comment:
Was this namespace change intentional? This doesn't feel like a demo because it's actually in the .SDK/ place (not the demos location)

Reply:
Hm, weird, yes - I don't remember changing that. Odd. So, which namespace are we using then? Stay with .Input?

Reply:
Yep, keep with what it was (unless we have a good reason to change it!)